src/index.tsx

```diff
@@ -23,6 +26,14 @@ initializeWindowExceptionHandler()
 //     iframe: false,
 //   })
 
+// Don't start the app machine until all these singletons
+// are initialized, and the wasm module is loaded.
+initPromise
+  .then(() => {
+    appActor.start()
+  })
+  .catch(reportRejection)
```

Comment:
Is this a general pattern we'll use for other actors?

I _believe_ this app actor should be the one responsible for starting the others.